notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2216-Slew_and_Track.ipynb

```diff
@@ -0,0 +1,827 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ccd559a-d0d4-4706-8ee2-2d5e78203346",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\"> Change configuration. </div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a836cfd-cd29-4e2b-aa3f-2392bef048d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.set_state(\n",
+    "    state=salobj.State.ENABLED,\n",
+    "    components=[\"mtm1m3\"], \n",
+    "    overrides = {\"mtm1m3\": 'Default'}\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19dcec67-b7a0-4a61-afc3-ceb1401e6725",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Ensure that we are not tracking\n",
+    "await mtcs.stop_tracking()\n",
+    "\n",
+    "# Make sure M1M3 is raised\n",
+    "await mtcs.raise_m1m3()\n",
+    "\n",
+    "# Make sure hardpoint corrections are enabled for M1M3\n",
+    "await mtcs.enable_m1m3_balance_system()\n",
+    "time.sleep(5)\n",
+    "\n",
+    "await mtcs.reset_m1m3_forces()\n",
+    "time.sleep(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29c406c2-94f9-4d35-9453-a63007acbc32",
+   "metadata": {},
+   "source": [
+    "### Get M2 LUT subscribed to the Mount\n",
+    "\n",
+    "- Put M2 in STANDBY\n",
+    "- Change the configuration files to make M2 LUT look at the mount elevation instead of the inclinometer.\n",
+    "- Reset the M2 EUI.\n",
+    "- Put M2 back in ENABLED state\n",
+    "- Turn on FB, and clear forces."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6c70bb89-dfd7-485b-baec-0637c29a03b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.set_state(\n",
+    "    state=salobj.State.STANDBY,\n",
+    "    components=[\"mtm2\"], \n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe7156eb-fc6c-4285-8f2a-40aea686c7d9",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\"> Change configuration. </div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a14e6374-5946-4690-8f0e-f4a371885468",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.set_state(\n",
+    "    state=salobj.State.ENABLED,\n",
+    "    components=[\"mtm2\"], \n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b56f74c-dcad-44aa-95e0-412dc988983c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make sure hardpoint corrections are enabled for M2\n",
+    "await mtcs.enable_m2_balance_system()\n",
+    "time.sleep(5)\n",
+    "\n",
+    "await mtcs.reset_m2_forces()\n",
+    "time.sleep(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e027820e-db41-4102-aa5a-50ec6f3e883e",
+   "metadata": {},
+   "source": [
+    "### Get CamHex ready\n",
+    "- Check config\n",
+    "- make sure LUT is on and has valid inputs\n",
```

Comment:
Same with the CamHex and M2Hex. How can we chech LUT is on, apart from actually slewing?

Reply:
The Hexapods do have an event that reports if the compensation mode is on or not. If it is on, you will get `evt_compensatedPosition` events. Otherwise, there are no events.